pom.xml

```diff
@@ -17,7 +17,16 @@
 	</properties>
 
 	<dependencies>
-
+		<dependency>
+			<groupId>net.sf.ehcache</groupId>
+			<artifactId>ehcache</artifactId>
+			<version>2.8.3</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context-support</artifactId>
+			<version>4.1.0.RELEASE</version>
```

Comment:
Use ${spring.version} to keep consistency with the rest of the spring dependencies